frontend/src/metabase/components/MetadataInfo/Popover/Popover.tsx

```diff
@@ -60,14 +58,19 @@ export function Popover({
       transitionProps={{
         duration: group.shouldDelay ? POPOVER_TRANSITION_DURATION : 0,
       }}
-      keepMounted
```

Comment:
This is the key fix. When we don't always render this component, we don't need to exclude it from `popover()` util anymore, since this will only appear after we hover over it.